### DISCUSSION TITLE
编辑器的包太大了，在React中使用怎么将编辑器搞成externals的方式，在运行时再去下载相应的包

### DISCUSSION THREAD

**Discussion by ChenKun1997** [wangeditor-team/wangEditor#4322] [Q&A] [answered]
如题，wangEditor的包太大了，我不想将他打包到bundle中，在React中使用怎么将这个WangEditor弄成externals的方式，在运行时再去下载相应的bundle，webpack的externals方式是这样的：https://webpack.docschina.org/configuration/externals/，但是我不知道如何配置

**Comment by wangfupeng1988** [comment, marked as answer]
这得去查 webpack 的文档

**Reply by ChenKun1997** [reply]
> 这得去查 webpack 的文档

好的，谢谢
